java/src/test/java/ai/rapids/cudf/TestExceptions.java

```diff
@@ -23,13 +23,15 @@
 public class TestExceptions {
   @Test
   public void testCudaCausedRmmException() {
-    assumeTrue(Cuda.isEnvCompatibleForTesting());
-    try {
-      Rmm.free(100, 0);
-      fail("An exception should have been thrown!!!");
-    } catch (RmmException rmme) {
-      Throwable cause = rmme.getCause();
-      assertTrue(cause instanceof CudaException);
-    }
+    // removing this for now, I expect this test to come back
```

Comment:
Why don't we just marked the test as `@Disabled` for now?

Reply:
fixed. That's much better. Thanks @jlowe